crates/api/src/deploy.rs

```diff
@@ -139,8 +129,13 @@ pub fn deploy(workdir: impl AsRef<Path>, path: impl AsRef<Path>) -> Result<()> {
     deploy_file.save(workdir)?;
     deploy_file.create_dirs(workdir)?;
 
-    log::info!("Deploy completed");
-    log::info!("Volumes = {:#?}", deploy_file.vols);
+    let res = deploy::DeployResult {
+        valid: Ok(Default::default()),
+        vols: deploy_file.vols,
+        start_mode: Default::default(),
+    };
+
+    println!("{}\n", serde_json::to_string(&res)?);
```

Comment:
```suggestion
    log::info!("{}\n", serde_json::to_string(&res)?);
```

Otherwise I get spurious prints when I didn't really ask for any.

Reply:
`deploy` needs to output this pure JSON to stdout. Either this or `deploy` returns a JSON for the cli to print.

Reply:
Hmm, could there be a way to do this in the CLI proper and not in the lib then?

Reply:
Done.